src/app/pages/my-virtrolio/my-virtrolio.component.html

```diff
@@ -71,20 +71,52 @@ <h5 class="modal-title" id="link-gen-title">Ahoy, your sharing link has been pre
           </div>
         </div>
 
-        <div class="row">
-          <label for="button-new-link" style="display:none">Click to generate a new link</label>
-          <button (click)="warnAndGenerate()" class="btn btn-secondary" id="button-new-link" type="button">Generate
-            new link
+        <div class="row justify-content-center" id="sharing-link-buttons">
+          <div *ngIf="canShare() else desktopSharing">
+            <label for="button-share-mobile">Share</label>
+            <button (click)="shareLink()" class="btn btn-success col-sm-5 mx-sm-1 my-2 my-sm-0" id="button-share-mobile"
+                    type="button">Share your link
+            </button>
+          </div>
+          <ng-template #desktopSharing>
+<!--            <label for="button-share-desktop">Share</label>-->
+<!--            <button (click)="openShareLinkSocialModal()" class="btn btn-info col-sm-5 mx-sm-1 my-2 my-sm-0"-->
+<!--                    id="button-share-desktop" type="button">Share your link-->
+<!--            </button>-->
+            <div class="row w-100" id="sharing-buttons">
```

Comment:
Add header - `ng serve` to see if it looks good first
```suggestion
            <div class="row w-100" id="sharing-buttons">
              <div><h6>Use the buttons below to share your link!</h6></div>
```